docs/recipes/pipelines.ipynb

```diff
@@ -709,7 +665,109 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Pipelines provide the benefit of removing a lot of cruft by taking care of tedious details for you. They also enable to clearly define what steps your model is made of. Finally, having your model in a single object means that you can move it around more easily. Note that you can include user-defined functions in a pipeline by using a `compose.FuncTransformer`."
+    "Pipelines provide the benefit of removing a lot of cruft by taking care of tedious details for you. They also enable to clearly define what steps your model is made of.\n",
+    "\n",
+    "Finally, having your model in a single object means that you can move it around more easily.\n",
+    "\n",
+    "Note that you can include user-defined functions in a pipeline by using a `compose.FuncTransformer`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Learning during predict\n",
+    "\n",
+    "In online macine learning, we can update the unsupervised parts of our model when a sample arrives. We don't _really_ have to wait for the ground truth to arrive in order to update unsupervised estimators that don't depend on it.\n",
+    "\n",
+    "In other words, in a pipeline, `learn_one` updates the supervised parts, whilst `predict_one` (or `predict_proba_one` for that matter) **can** update the unsupervised parts. \n",
```

Comment:
```suggestion
    "In other words, in a pipeline, `learn_one` updates the supervised parts, whilst `predict_one` (or `predict_proba_one` for that matter) **can** update the unsupervised parts, which often yields better results. \n",
```

If anyone has the motivation, it could actually be nice to also write a new notebook in the examples section of the docs illustrating the potential performance difference from using `learn_during_predict` on a suitable dataset.

Could be a good first issue.

Reply:
I will try to integrate a sort of _comparison_ section

Reply:
Don't bother too much with that here if it gets complex. The goal is also to have something easily digestible to new users

Reply:
Implemented code to obtain [this result](https://github.com/online-ml/river/pull/1528#discussion_r1573231480).
Not sure if you had something in particular in in mind, but I was curious and tried such experiment.